grpc/rust/BUILD

```diff
@@ -36,30 +38,42 @@ cargo_build_script(
         "//common:transaction-proto",
         "//core:database-proto",
         "//core:service-proto",
-        "@com_google_protobuf//:protoc",
-    ],
-    build_script_env = {
-        "PROTOC": "$(execpath @com_google_protobuf//:protoc)"
-    },
-    deps = [
-        "@vaticle_dependencies//library/crates:tonic_build"
-    ],
+    ]
 )
 
 rust_library(
     name = "typedb_protocol",
-    srcs = ["lib.rs"],
-    deps = [
+    srcs = [
         ":typedb_protocol_src",
+        "lib.rs",
+    ],
+    deps = [
         "@vaticle_dependencies//library/crates:tonic",
         "@vaticle_dependencies//library/crates:prost",
     ],
     tags = [
         "crate-name=typedb-protocol",
     ],
+    version = "0.1.2",
```

Comment:
We released version `0.1.2` manually, so there's no need to re-release this. However, we should add the CI release scripts.